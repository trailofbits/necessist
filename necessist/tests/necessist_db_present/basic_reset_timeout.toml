args = ["--root=fixtures/basic", "--reset", "--timeout=5"]
stderr = ""

[bin]
name = "necessist"

[fs]
cwd = "../../.."

[status]
code = 0
