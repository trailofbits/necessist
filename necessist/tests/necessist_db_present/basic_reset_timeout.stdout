4 candidates in 4 tests in 1 test file
fixtures/basic/src/lib.rs: dry running
fixtures/basic/src/lib.rs: mutilating
fixtures/basic/src/lib.rs:4:5-4:12: `n += 1;` passed
